features/login/impl/build.gradle.kts

```diff
@@ -19,7 +19,7 @@ plugins {
     alias(libs.plugins.anvil)
     alias(libs.plugins.ksp)
     id("kotlin-parcelize")
-    kotlin("plugin.serialization") version "1.9.20"
+    kotlin("plugin.serialization") version libs.versions.kotlin
```

Comment:
```suggestion
    alias(libs.plugins.kotlin.serialization)
```

Reply:
Since it's using `kotlin` (and not `id`), are you sure this is equivalent?

Reply:
Pretty sure it's equivalent, the decompiled code just prepends "org.jetbrains.kotlin." to the argument:

```
   @NotNull
   public static final PluginDependencySpec kotlin(@NotNull PluginDependenciesSpec $this$kotlin, @NotNull String module) {
      Intrinsics.checkNotNullParameter($this$kotlin, "<this>");
      Intrinsics.checkNotNullParameter(module, "module");
      PluginDependencySpec var10000 = $this$kotlin.id("org.jetbrains.kotlin." + module);
      Intrinsics.checkNotNullExpressionValue(var10000, "id(\"org.jetbrains.kotlin.$module\")");
      return var10000;
   }
```